packages/frontend/src/pages/create-with-template-id/index.tsx

```diff
@@ -121,25 +143,51 @@ export const CreateWithTemplateId = () => {
 
     return (
         <Layout navbarBgColor="green" noMarquee>
-            <div className="flex-grow bg-grid-light bg-left-top bg-green">
-                {!pinningProxyAuthenticated ? (
+            <div className="h-screen flex-grow bg-grid-light bg-left-top bg-green">
+                {!creatorAllowed ? (
```

Comment:
Fixed in 339fa484d9f180c88d1b0628cffb5857431cbccf.

Reply:
What happens if loading is true but `creatorAllowed` is false too, which I'm guessing it's its default state? I think the `!creatorAllowed` check should be after the loading action for it is what I mean

Reply:
Oh you're right, fixed in ab18a4b5629eadad8d2bd959c8284e03638be937. Moved the loading state to the top, otherwise the `Authenticate` would show up "before" the permissioned warning.